src/components/__tests__/data-table.spec.jsx

```diff
@@ -8,7 +8,7 @@ describe('DataTable', () => {
   const onSelect = jest.fn();
   const onHeaderClick = jest.fn();
   const selectable = true;
-  const selected = {};
+  const selected = { id0: true };
```

Comment:
I suppose `id0` is NOT a typo and the proper format for the index, right?

Reply:
correct, see below